src/cmd/tools/read_data_files/main/main.go

```diff
@@ -111,9 +111,8 @@ func main() {
 		log.Fatalf("unknown benchmark type: %s", *benchmark)
 	}
 
-	bytesPool := tools.NewCheckedBytesPool()
-	bytesPool.Init()
-
+	// Not using bytes pool with streaming reads/writes to avoid the fixed memory overhead.
+	var bytesPool pool.CheckedBytesPool
```

Comment:
Perhaps don’t bother creating the variable and instead remove the following calls to SetBytesPool(..)?

Reply:
Ups, automerged.
I left this variable there intentionally because it is also used slightly below:
```go
reader, err := fs.NewReader(bytesPool, fsOpts)
```
My thought was that we often implement derived versions of this tool with some added functionality, and being more clear about where the pool might be used might potentially help in some situations (eg. if we need to back out of streaming reads).